configs/atss/README.md

```diff
@@ -0,0 +1,24 @@
+# Bridging the Gap Between Anchor-based and Anchor-free Detection via Adaptive Training Sample Selection
+
+
+## Introduction
+
+```
+@article{zhang2019bridging,
+  title   =  {Bridging the Gap Between Anchor-based and Anchor-free Detection via Adaptive Training Sample Selection},
+  author  =  {Zhang, Shifeng and Chi, Cheng and Yao, Yongqiang and Lei, Zhen and Li, Stan Z.},
+  journal =  {arXiv preprint arXiv:1912.02424},
+  year    =  {2019}
+}
+```
+
+
+## Results and Models
+
+| Backbone  | Style   | Lr schd | Mem (GB) | Train time (s/iter) | Inf time (fps) | box AP |
+|:---------:|:-------:|:-------:|:--------:|:-------------------:|:--------------:|:------:|
+| R-50      | pytorch | 1x      |          |                     |                |  38.8  |
```

Comment:
The official box AP is 39.3, 0.5 AP difference is a large gap. It is necessary to find out where the difference comes.

Reply:
we have changed the code according to the `Changes requested` and retrain the model(we train the model use 2 GPUs, to speed up training we set `imgs_per_gpu=4` and  `lr = 0.01 / 2`), now the ap is 39.4, here is the [train log](https://drive.google.com/open?id=1KlPSss2OKpfmWVmKWPxtmKnT0fnGYlFT)



we doubt the difference may caused by one of following reasons

- whether use `all reduce` to compute `num_total_samples` and `bbox_avg_factor`
- when we  train the old model, our server power off during 6th epoch, so we resume training from 5th epoch. theoretically there is no problem, but when we compared the ap of old and retrained model at each epoch, we find that there is a large gap after 5th epoch, and for old model, the ap of 6th epoch is equal ap of 5th epoch, so we doubt that there may be something error when we resume training 

| model | epoch_1 | epoch_2 | epoch_3 | epoch_4 | epoch_5 | epoch_6 | epoch_7 | epoch_8 | epoch_9 | epoch_10 | epoch_11 | epoch_12|
|:-----:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:--------:|:--------:|:------:|
| `old `|  0.216  |  0.255  |  0.286  |  0.296  |  0.310  |  0.310  |  0.317  |  0.326  |  0.379  |  0.383   |  0.385   |  0.388   |
| `new` |  0.213  |  0.257  |  0.287  |  0.303  |  0.309  |  0.326  |  0.327  |  0.331  |  0.386  |  0.388   |  0.391   |  0.394   |